shared/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+TODO: Can this be a patch release?
```

Comment:
TODO: I think patch is fine here? But I'm not sure.

Reply:
Eh, I'd just go with minor. It doesn't really hurt.